Packs/XForceExchange/Integrations/XFE_v2/XFE_v2.yml

```diff
@@ -43,7 +43,7 @@ configuration:
   type: 0
 description: IBM X-Force Exchange lets you receive threat intelligence about applications,
   IP addresses, URls and hashes
-display: IBM X-Force Exchange (v2)
+display: IBM X-Force Exchange v2
```

Comment:
as long as you fix the integration, please update line 58 in this file (remove trailing space and fix typo):
should be `description: score threshold`